guides/common/modules/proc_enabling-pull-mode-on-proxy.adoc

```diff
@@ -0,0 +1,23 @@
+[id="enabling-pull-mode-on-proxy_{context}"]
```

Comment:
I think this is sufficiently covered by https://github.com/theforeman/foreman-documentation/pull/1548 so this file can be removed.